### DISCUSSION TITLE
MDX 2 support in next major Prettier version

### DISCUSSION THREAD

**Discussion by kachkaev** [mdx-js/mdx#2127] [General]
Hey folks! 

As some of you may have heard, Prettier team is working on releasing v3:

- [checklist](https://github.com/prettier/prettier/issues/13142)
- [milestone](https://github.com/prettier/prettier/milestone/14)
- [blog post draft](https://github.com/prettier/prettier/pull/13436)

Prettier v2 is still on MDX 1 and MDX 2 support is tracked in https://github.com/prettier/prettier/issues/12209. There has been little progress on in for about six months, so I wonder if anyone from MDX community could help us with upgrading.

I guess that Prettier v3 will be released somewhat soon regardless of MDX 2 status because other important breaking changes are pending. Missing v3 milestone will be a shame for MDX 2 because the next window of opportunity will probably be Prettier v4 only.

If anyone has spare capacity, please get involved and help us! 🙏  If you decide to do dev work, please leave a comment in https://github.com/prettier/prettier/issues/12209 to avoid duplicate effort.

**Comment by JounQin** [comment]
I'll try to work on it soon.

**Comment by clicktodev** [comment]
mdx 3 support would be appreciated
